legion/engine/physics/physx/physx_event_process_funcs.cpp

```diff
@@ -168,7 +180,10 @@ namespace legion::physics
             PxTransform localTransform;
             calculateLocalColliderTransform(localTransform, convex);
 
-            PxConvexMesh* convexMesh = static_cast<PxConvexMesh*>(convex.getConvexPtr().ptr);
+            //const cast because PxConvexMeshGeometry does not accept const PxConvexMesh*
+            const PxConvexMesh* cconvexMesh = static_cast<const PxConvexMesh*>(convex.getConvexPtr().ptr);
+            PxConvexMesh* convexMesh = const_cast<PxConvexMesh*>(cconvexMesh);
```

Comment:
if you have to const cast it, then arguably it shouldn't be const to begin with?